k8s/cmd/nsmd-k8s/main.go

```diff
@@ -37,33 +33,19 @@ func main() {
 
 	address := os.Getenv("NSMD_K8S_ADDRESS")
 	if strings.TrimSpace(address) == "" {
-		address = "127.0.0.1:5000"
+		address = "0.0.0.0:5000"
```

Comment:
I hope that this will be better secured with our "security" coming.
But I wonder why it is needed in the first place. Interesting to see what the architecture is.

Reply:
It will be changed with realizing nsmd-k8s as plugin